### ISSUE TITLE
Issues with displaying Shadow DOM

### ISSUE THREAD

**Issue by levinsxwarp** [storybookjs/react-inspector#187]
I am currently working with the react-inspector package and I encountered an issue when trying to inject a Shadow DOM into the page. Specifically, I am trying to create a shadowRoot and inject a shadow host element into the DOM. While the host element itself appears as expected, the shadow DOM’s root element and its child elements are not rendered within the page.

![Image](https://github.com/user-attachments/assets/354b0c57-11c2-4dae-a279-04a02d9d8c58)

**Comment by Sidnioulz** [comment]
I don't think we plan on adding support for new things in this package. We would need outside help to build support for Shadow DOM.
